shop/shop/src/main/java/ro/msg/learning/shop/model/ProductCategory.java

```diff
@@ -0,0 +1,27 @@
+package ro.msg.learning.shop.model;
+import lombok.Data;
+import javax.persistence.Entity;
+import javax.persistence.OneToMany;
+import java.util.Set;
+
```

Comment:
It's enough to have a separator of 1 line, there is no need to have 2 or more anywhere